src/SDKs/Azure.Base/data-plane/Azure.Base/Pipeline/ResponseClassifier.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+
+namespace Azure.Base.Pipeline
+{
+    public abstract class ResponseClassifier
```

Comment:
I am not thrilled by the name. "Classifier" is such a grandiose name (more fitting the ML space).

Reply:
I'm fine with whatever name, I just don't have a better idea, unfortunately.